tests/rados/pool_tests.py

```diff
@@ -511,26 +511,36 @@ def run(ceph_cluster, **kw):
         # Now that the autoscaler has been turned on, the warnings should go and PGs should be scaled accordingly
         time.sleep(300)
 
-        flag = True
-        end_time = datetime.datetime.now() + datetime.timedelta(seconds=2000)
+        alert_flag = False
+        end_time = datetime.datetime.now() + datetime.timedelta(seconds=4000)
```

Comment:
Maybe we should reduce the wait time to 15 mins and monitor few runs, we can increase the wait time later on if observe false failures.

Reply:
Updated.